barcodes_gs1_label/views/report_barcode.xml

```diff
@@ -29,7 +16,7 @@
             <div class="bgp-barcode">
                 <div class="bgp-barcode-img">
                     <img
-                        t-att-src="'/report/barcode/?type=%s&amp;value=%s&amp;height=%s' % (barcode_type, wc[1]['barcode_string'], 100)"
+                        t-att-src="'/report/barcode/?barcode_type=%s&amp;value=%s&amp;height=%s' % (barcode_type, wc[1]['barcode_string'], 100)"
```

Comment:
@deeniiz this was the issue we discuss yesterday. Now it returns the barcode image